tests/robustness/traffic/etcd.go

```diff
@@ -72,6 +72,21 @@ type etcdTraffic struct {
 	largePutSize int
 }
 
+func (t etcdTraffic) WithoutRevision() Traffic {
```

Comment:
why is this called `WithoutRevision` while it seems to be `WithoutCompact`?

Reply:
Ups :P